librz/include/rz_cons.h

```diff
@@ -1200,6 +1215,11 @@ RZ_API int rz_line_set_hist_callback(RzLine *line, RzLineHistoryUpCb cb_up, RzLi
 RZ_API int rz_line_hist_cmd_up(RzLine *line);
 RZ_API int rz_line_hist_cmd_down(RzLine *line);
 
+RZ_API int rz_line_undo_reset(void);
+RZ_API int rz_line_undo_add_entry(RzLineUndoType type, int start, int end, char *text);
+RZ_API int rz_line_undo_begin_action();
+RZ_API int rz_line_undo_end_action();
```

Comment:
it is not clear when/how we should use these... why are they necessary? Please add doc for these 4 functions and also some doc explaining the meaning and usage of the RzLineUndoType.

Reply:
these should be `RZ_IPI` and these needs to be moved under `core_private.h`

Reply:
I will remove these functions.